mungegithub/submit-queue/deployment.yaml

```diff
@@ -16,6 +16,7 @@ spec:
         - --token-file=/etc/secret-volume/token
         - --pr-mungers=blunderbuss,lgtm-after-commit,cherrypick-auto-approve,label-unapproved-picks,needs-rebase,ok-to-test,rebuild-request,path-label,size,stale-pending-ci,stale-green-ci,block-path,release-note-label,comment-deleter,submit-queue
```

Comment:
Oh, I'm a bad person and I changed the list in prod without checking it in. If we're keeping this file up to date, can you add issue-cache and flake-manager?